### DISCUSSION TITLE
HTML's orginal style lost!

### DISCUSSION THREAD

**Discussion by tianPenga** [tailwindlabs/tailwindcss#13360] [Help]
using @tailwindcss base made my html's orginal style miss?why would happen this?

**Comment by alex-krasikau** [comment]
Hey @tianPenga!

Tailwind CSS includes Preflight by default, which is a set of base styles that standardize behavior across browsers. You can access all these styles in this file: [Preflight Styles](https://unpkg.com/tailwindcss@%5E3/src/css/preflight.css).

To turn off this feature, add `preflight: false` to your Tailwind CSS configuration. Below is the updated configuration:
```js
module.exports = {
  corePlugins: {
    preflight: false,
  }
}
```

**Comment by tianPenga** [comment]
It works, so grateful!



---- Replied Message ----
| From | ***@***.***> |
| Date | 03/26/2024 18:04 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [tailwindlabs/tailwindcss] HTML's orginal style lost! (Discussion #13360) |

Hey @tianPenga!

Tailwind CSS includes Preflight by default, which is a set of base styles that standardize behavior across browsers. You can access all these styles in this file: Preflight Styles.

To turn off this feature, add preflight: false to your Tailwind CSS configuration. Below is the updated configuration:

module.exports={corePlugins: {preflight: false,}}
